### ISSUE TITLE
March 14th - Time series data of previous dates are now wrong

### ISSUE THREAD

**Issue by calvin-falcon** [CSSEGISandData/COVID-19#777]
Yesterday's data was fine. When someone committed the new confirmed cases to the time series data today, they change the previous day's values and they are now wrong.

For example, in many countries we now see the same number of reported cases repeated for 2 days or more, multiple times. I suggest reverting to the table from yesterday and just adding the days for March 14th.

**Comment by JiPiBi** [comment]
Sorry, but I have not the same observations on the countries I follow (apart for the 12th that was eroneous and not corrected for the moment ) , could you show some examples to illustrate

**Comment by calvin-falcon** [comment]
Yes sure. So far I've only seen in in Portugal and assumed there could be other duplications for other countries. 

Examples of that for the dates of 03/08/2020 + 03/09/2020 and 03/11/2020 + 03/12/2020. The new csv has the number of confirmed cases repeated for each pair of dates (30, 30 AND 59 , 59) where the old one was accurate (30, 39 AND 59 78).

**Comment by JiPiBi** [comment]
You must be right 
You gan also see this link : https://qap.ecdc.europa.eu/public/extensions/COVID-19/COVID-19.html
And there is  another link to download the data of this dashboard:   https://www.ecdc.europa.eu/en/publications-data/download-todays-data-geographic-distribution-covid-19-cases-worldwide

**Comment by calvin-falcon** [comment]
Thank you! Also I really appreciate the work put in this project. Thank you!

**Comment by imoretti** [comment]
I also see all the US county/state data is zeros for the last four days. :(

**Comment by Sitius86** [comment]
Folks- Please take 30 seconds to skim through some of the other issues before posting, many of these have already been discussed at length and some even formally responded to by the JHU team directly. The US count/state data will no longer be updated, as it was becoming too difficult to manage.  Cheers
